templates/course-results.php

```diff
@@ -72,7 +72,7 @@
 
 			<section class="course-results-lessons">
 				<?php
-				if ( Sensei_Course::is_user_enrolled( $course->ID ) ) {
+				if ( Sensei_Utils::has_started_course( $course->ID ) ) {
```

Comment:
Looks like we'll need to pass the user ID here. [Otherwise it will use 0](https://github.com/Automattic/sensei/blob/ea76088d90285705814c38e31fb0d4f11705e282/includes/class-sensei-utils.php#L1477).

Reply:
Yup that can cause some problems on the filters hooked onto it. Fixing in #2978.